docs/source/utils/generate_tutorials.py

```diff
@@ -129,7 +135,7 @@ def generate_tutorial_links_for_notebook_creation(
         else:
             flattened += [f"{package[0]}.{subpackage[0]}" for subpackage in package[2]]
 
-    links = iterate_tutorials_dir_generating_links(Path(source), dest, source)
+    links = iterate_tutorials_dir_generating_links(Path(source), dest, f"{destination}/tutorials")
```

Comment:
Because when several versions are being built at the same time, I had to create tutorial links within temporary directories and not the original source directory of the branch/tag we're in. (or tutorials would start intersecting in that directory) Currently there is only one version built at a time, so this could be reverted, but in case there is ever a need to build several versions at a time, this is already configured for that.
Also, the iterate_tutorials_dir_generating_links() function utilised the fact that source="tutorials" so that they could speed up the naming process, which doesn't work in case source="tmp_dir/tutorials", which it is in case of building for many versions. Because then I'd have "tmp_dir/docs/source/tutorials/tmp_dir/tutorials.tutorial_name" as the destination path for create_notebook_link().
Most other changes here are things that I forgot to remove, like adding an unused parameter.